Add QuTiP to compare QHSO and the transmon (#1790)
qutip/qutip: README.md

[![PyPi Downloads](https://img.shields.io/pypi/dm/qutip?label=downloads%20%7C%20pip&logo=PyPI)](https://pypi.org/project/qutip)
[![Conda-Forge Downloads](https://img.shields.io/conda/dn/conda-forge/qutip?label=downloads%20%7C%20conda&logo=Conda-Forge)](https://anaconda.org/conda-forge/qutip)

> QuTiP is an open-source software for simulating the dynamics of closed and open quantum systems.
> The QuTiP library uses Numpy, Scipy, and Cython packages as numerical backends, and Matplotlib for graphical output.

Reviewer: ```suggestion
QuTiP is open-source software for simulating the dynamics of closed and open quantum systems.
It uses the excellent Numpy, Scipy, and Cython packages as numerical backends, and graphical output is provided by Matplotlib.
```
"software" is an uncountable noun and so does not take an article. Praising numpy, SciPy and Cython shows we're grateful to those projects and sets a good tone, so I'd like to keep it. Using a slightly different phrasing when mentioning Matplotlib makes the sentence less repetitive, so I'd like to keep it that way.

QuTiP aims to provide user-friendly and efficient numerical simulations of a wide variety of quantum mechanical problems, including those with Hamiltonians and/or collapse operators with arbitrary time-dependence, commonly found in a wide range of physics applications.
QuTiP is freely available for use and/or modification, and it can be used on all Unix-based platforms and on Windows.

> QuTiP does not charge any licensing fees and hence is ideal for exploring quantum mechanics in research as well as in the classroom.

Reviewer: ```suggestion
Being free of any licensing fees, QuTiP is ideal for exploring quantum mechanics in research as well as in the classroom.
```
I think the new phrasing is a slight degradation, so I'd like to keep the original.
